.github/workflows/index-packages.yml

```diff
@@ -0,0 +1,89 @@
+name: Index packages in Azure/incoming
+
+
+on:
+  workflow_call:
+    inputs:
+      pkg-type:
+        required: true
+        type: string  # stable / nightly
```

Comment:
[Note]
I would have suggested using `type: choice` but currently it is only supported in manually dispatched workflows :/

Reply:
Yep, it would be nice to have that :(